README.adoc

```diff
@@ -6,6 +6,91 @@ image:https://codecov.io/gh/codeready-toolchain/toolchain-common/branch/master/g
 
 This repo is for controllers, libs, scripts, make files, etc to be shared between host and member operators.
 
+== Setting Up and Connecting Host and Member Clusters
+
+In a new terminal, execute the following commands
+```
+$ cd $GOPATH/github.com/codeready-toolchain/host-operator
+$ minishift start --profile host
+$ make login-as-admin 
+$ make create-namespace 
+$ make deploy-rbac
+$ make deploy-crd
+```
+
+```
+$ cd $GOPATH/github.com/codeready-toolchain/member-operator
+$ minishift start --profile member
+$ make login-as-admin 
+$ make create-namespace 
+$ make deploy-rbac
+$ make deploy-crd
+$ make add-member-to-host
+$ make add-host-to-member
+$ make build
+$ make vendor
+$ export OPERATOR_NAMESPACE=toolchain-member-operator
+$ operator-sdk up local --namespace=toolchain-member-operator --verbose
+```
+
+Now that we are setup, we can add resources and watch the logs on the member-operator! To do so, open a new terminal and run the following commands
+```
+$ cd $GOPATH/github.com/codeready-toolchain/host-operator
+$ minishift profile set host
+$ make use-namespace
+$ oc apply -f '/path/to/masteruserrecord.yaml'
+$ cd $GOPATH/github.com/codeready-toolchain/member-operator
+$ minishift profile set member
+$ make use-namespace
+$ oc apply -f '/path/to/useraccount.yaml'
+```
+
+**Example masteruserrecord.yaml:**
+
+_Note:_ The `targetCluster` field must contain the correct IP address of the member cluster.
+```
+apiVersion: toolchain.dev.openshift.com/v1alpha1
+kind: MasterUserRecord
+metadata:
+  name: example
+spec:
+  disabled: false
+  deprovisioned: false
+  userID: 86505192-a386-11e9-ad56-525400ad2b23
+  userAccounts:
+  - targetCluster: member-192-168-42-61-8443
```

Comment:
This should actually be the _name_ of the target cluster, not the IP address.  We use the name to lookup the cluster info, and in fact there is a function (see [here](https://github.com/codeready-toolchain/toolchain-common/blob/master/pkg/cluster/cache.go#L67)) that accepts the cluster name, which is used to get a client for the cluster.

Reply:
@sbryzak Thanks for clarifying! I have updated the comment to describe the field as the **name**. I see in your PR this field contains the name of the cluster we would be pointing at: https://github.com/codeready-toolchain/host-operator/pull/27/files#diff-1696f26dd78d9fa4906ecb5d5c0911c3R7. I guess this looks a bit different since it is using minishift.